### ISSUE TITLE
Cloud ressource manager folder

### ISSUE THREAD

**Issue by antoine-castex** [googleapis/google-cloud-python#6377]
hello

i need to have the name of a google cloud folder with this method

https://cloud.google.com/resource-manager/reference/rest/v2/folders/get

impossible to find them in the python client library :(

**Comment by tseaver** [comment]
@antoine-castex You are correct that we do not support the V2 `folders.get` feature:  in fact, we do not support the V2 API at all, nor even V1 (see #6039).
